src/index.js

```diff
@@ -123,13 +123,19 @@ class ChatGPT {
 	}
 
 	async getTokens() {
-		await this.Wait(1000);
+		await this.wait(1000);
 		let data = await new Promise((resolve) => {
 			this.socket.emit("getSession", this.sessionToken, (data) => {
 				resolve(data);
 			});
 		});
-		if (data.error) console.log(`Error: ${data.error}`);
+		if (data.error) {
+			if (!this.auth) {
```

Comment:
why you checked for `this.auth` here?
I think not needed because the first time `this.auth` is null and it will definitely throw if an error happens because `this.auth` is set below the check

Reply:
Yeah, that's the point. It's so that if you start the app with an invalid token, it'll throw an error. I thought it might help reduce confusion but I can revert that if you want.

Reply:
no, it's ok, well done